recipes/gitlab-runner/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set import_path = "gitlab.com/gitlab-org/gitlab-runner" %}
+{% set version = "12.1.0" %}
+{% set branch = '-'.join(version.split('.')[:2] +['stable']) %}
+
+{% set name = import_path.split('/')[-1] %}
+{% set pkg_src = ('src/'+import_path).replace("/", os.sep) %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  - folder: {{ pkg_src }}
+    url: https://{{ import_path }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.bz2
+    sha256: 49e7993e1eabc362ec99562e84931e699b77948efb6669f3e98a5dc40f2b2b66
+
+build:
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    #- for /F "delims=" %%i in ('date.exe /t +%Y-%m-%dT%H:%M:%S%z') DO set BUILT="%%i"  # [win]
+    - set BUILT="2019-08-14T00:00:00+0000"  # [win]
+    - export BUILT=$(date -u +%Y-%m-%dT%H:%M:%S%z)  # [unix]
+    {% set flags = dict(VERSION=version, REVISION="deadbeef", BRANCH=branch, BUILD="%BUILT%" if win else "$BUILT") %}
```

Comment:
Guessing from build log: the key should be BUILT instead if BUILD.

Reply:
I thought so as well, until I checked upstream:

https://gitlab.com/gitlab-org/gitlab-runner/blob/master/Makefile#L51